app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-list/dynamic-content-form-field-list.html

```diff
@@ -64,14 +64,15 @@
           schema="schema"
           ng-model="itemValues"
           on-validity-change="isItemInvalid = internalForm.$dirty && !isValid"
-          class="accordion-list-item-content">
+          class="accordion-list-item-content"
+          hide-label>
       </dynamic-content-form-field-object>
 
       <div class="accordion-list-item-meta">
 
         <div class="accordion-list-item-meta-tools">
           <button
-              class="btn btn-xs btn-danger"
+              class="btn btn-sm btn-danger"
```

Comment:
This button looked super cramped and now matches [the style of live blog list](https://github.com/theonion/bulbs-cms/blob/master/app/components/live-blog/live-blog-entries/live-blog-entries.html#L149).